LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -171,7 +171,7 @@ public async Task<Message> ReceiveAsync(TimeSpan timeout)
 
                 return msg;
             }
-            catch (OperationCanceledException ex) when (ExceptionFilterUtility.True(() => this.logger.LogError($"could not retrieve cloud to device message with error: {ex.Message}")))
+            catch (OperationCanceledException) when (ExceptionFilterUtility.True(() => this.logger.LogError("could not retrieve cloud to device message due to timeout.")))
```

Comment:
pls always pass the original exception to the logger. The stack traces where the timeout happened can be crucial.